portfolio/src/main/webapp/script.js

```diff
@@ -71,6 +71,7 @@ document.addEventListener('DOMContentLoaded', () => {
   const imgAnchor = new ImageAnchors();
   document.albumSlide.src = images[0].filePath;
   document.getElementById('wrapper').href = images[0].link;
+  getComments();
```

Comment:
Fun idea and we can talk about this later, you can set things up so you don't have to wait for the DOM to be loaded to start the fetch. BUT it means you'll have to wait for the DOM to be loaded to handle the response (since you will be adding to the element!).

You don't have to handle this case at all, but I present it as a challenge (and I'll suggest it to the rest of the pod to).

Reply:
Would this mean to have getComments() outside of any event listener and to create an event listener for when the response is ready?